src/explore-education-statistics-admin/src/pages/bau/BoundaryDataPage.tsx

```diff
@@ -61,39 +61,33 @@ const BoundaryDataPage = () => {
           </dl>
         </Details>
 
-        <Details
-          summary="New data uploads: before you start"
-          id="beforeYouStart"
-        >
-          <p>
-            If the GeoJSON file has been downloaded direcly from the Open
-            Geography Portal then the coordinate system may need converting from
-            EPSG:27700 to EPSG:4326 (WGS84) before using this tool. The file can
-            be converted using "mapshaper", following the below steps:
-          </p>
-          <ol>
-            <li>
-              Visit{' '}
-              <a href="https://mapshaper.org/" target="_blank" rel="noreferrer">
-                https://mapshaper.org/
-              </a>
-            </li>
-            <li>Import the original GeoJSON file</li>
-            <li>Select "Console" from the navigation options</li>
-            <li>
-              Enter "<code>-proj from=EPSG:27700 crs=EPSG:4326</code>" and press
-              ENTER
-            </li>
-            <li>Select "Export" from the navigation options</li>
-            <li>
-              Ensure the "GeoJSON" option is selected, then click "Export" to
-              save the file to your local machine
-            </li>
-            <li>
-              Rename the downloaded file's extension from .json to .geojson
-            </li>
-          </ol>
-        </Details>
+        <h2>New data uploads: before you start</h2>
```

Comment:
Should this section be using `InsetText` to make the styling consistent with other 'Before you start' sections? e.g.
![Screenshot 2024-07-16 103738](https://github.com/user-attachments/assets/c818dddf-4c82-4a0a-a4ef-e5acde23bc3b)